openapi-integration-tests/src/test/resources/client/expected/project-09/types.bal

```diff
@@ -69,7 +69,7 @@ public type chatCompletionsRequestCommon record {
     # If set, partial message deltas will be sent, like in ChatGPT. Tokens will be sent as data-only server-sent events as they become available, with the stream terminated by a `data: [DONE]` message.
     boolean? 'stream = false;
     # Up to 4 sequences where the API will stop generating further tokens.
-    string|string[]? stop = "null";
+    string|string[]? stop = null;
```

Comment:
In here shouldn't we do
```suggestion
    string|string[]? stop = ();
```
?

Reply:
This is generated from the default value provided in the OpenAPI specification. We can special case this value and convert it to `()`. Since [the best practice is to use `()` instead of `null`](https://learn-ballerina.github.io/best_practices/represent_optionality.html?highlight=null#represent-optionality), will fix this 👍🏼

Reply:
[b4ab821](https://github.com/ballerina-platform/openapi-tools/pull/1767/commits/b4ab8212e6b362bd2f3efa43c0a54b19143a63ca)